vlasiator.cpp

```diff
@@ -992,7 +992,14 @@ int main(int argn,char* args[]) {
             logFile << "(AMR): Adapting refinement!"  << endl << writeVerbose;
             if (!adaptRefinement(mpiGrid, technicalGrid, sysBoundaryContainer, *project)) {
                // OOM, rebalance and try again
+               for (auto id : mpiGrid.get_local_cells_to_refine()) {
+                  mpiGrid[id]->parameters[CellParams::LBWEIGHTCOUNTER] *= 8.0;
+               }
                balanceLoad(mpiGrid, sysBoundaryContainer);
+               for (auto id : mpiGrid.get_local_cells_to_refine()) {
```

Comment:
Now this has me wondering. Should we in fact leave the LBWEIGHTCOUNTER value intact? Because then it would write those values into the restart file, and they would be used by the new more-tasks run for initial LB!

Reply:
And so this is handled now, correct?

Reply:
Yes, in a different way